code/_macros.dm

```diff
@@ -79,8 +79,6 @@
 
 #define attack_animation(A) if(istype(A)) A.do_attack_animation(src)
 
-#define isairlock(A) istype(A, /obj/machinery/door/airlock)
```

Comment:
don't remove this, it'll be used in future, surprised its not already. There are plenty of places it could be

Reply:
this is duplicated.